### ISSUE TITLE
Departure,Arrival date selection

### ISSUE THREAD

**Issue by pashishraj** [Mandal-21/Flight-Price-Prediction#11]
Arrival date shouldn't be less than departure date.

**Comment by Mandal-21** [comment]
Yes. You can put a check on that, if a person is selecting a date and time for Arrival before departure date and time, you can through an error.

**Comment by madhumidhaaa** [comment]
> Yes. You can put a check on that, if a person is selecting a date and time for Arrival before departure date and time, you can through an error.

**Comment by madhumidhaaa** [comment]
Its not throwing any error....the amount is being calculated even if the arrival time is before departure time
